tests/framework/extensions/provisioning/creates.go

```diff
@@ -594,6 +600,81 @@ func CreateProvisioningRKE1AirgapCustomCluster(client *rancher.Client, clustersC
 	return createdCluster, err
 }
 
+// CreateProvisioningAKSHostedCluster provisions an AKS cluster, then runs verify checks
+func CreateProvisioningAKSHostedCluster(client *rancher.Client) (*management.Cluster, error) {
+	cloudCredential, err := azure.CreateAzureCloudCredentials(client)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterName := namegen.AppendRandomString("akshostcluster")
+	clusterResp, err := aks.CreateAKSHostedCluster(client, clusterName, cloudCredential.ID, false, false, false, false, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	if client.Flags.GetValue(environmentflag.UpdateClusterName) {
+		pipeline.UpdateConfigClusterName(clusterName)
+	}
+
+	client, err = client.ReLogin()
+	if err != nil {
+		return nil, err
+	}
+
+	return client.Management.Cluster.ByID(clusterResp.ID)
+}
+
+// CreateProvisioningEKSHostedCluster provisions an EKS cluster, then runs verify checks
+func CreateProvisioningEKSHostedCluster(client *rancher.Client) (*management.Cluster, error) {
+	cloudCredential, err := aws.CreateAWSCloudCredentials(client)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterName := namegen.AppendRandomString("ekshostcluster")
+	clusterResp, err := eks.CreateEKSHostedCluster(client, clusterName, cloudCredential.ID, false, false, false, false, nil)
```

Comment:
not for our team, but highlanders should have a ticket to extend this to allow user-entered labels, etc.